fastn-core/src/manifest/mod.rs

```diff
@@ -53,17 +53,17 @@ pub async fn write_manifest_file(
     );
 
     let zip_url = match zip_url {
-        Some(zip_url) => zip_url.to_string(),
+        Some(zip_url) => Some(zip_url.to_string()),
         None => {
             match fastn_core::manifest::utils::get_gh_zipball_url(config.package.name.clone()) {
-                Some(gh_zip_url) => gh_zip_url,
+                Some(gh_zip_url) => Some(gh_zip_url),
                 None => {
-                    return Err(fastn_core::error::Error::UsageError {
-                        message: format!(
-                            "Could not find zip url for package \"{}\".",
-                            &config.package.name
-                        ),
-                    })
+                    fastn_core::warning!(
+                        "Could not find zip url for package \"{}\".",
+                        &config.package.name
+                    );
+
```

Comment:
This is not a warning. Would everything work if this happens? If not let user know, what's the point of returning success and failing?

Reply:
@harshdoesdev this is pending.